docker/rbac.providers.ldap.Dockerfile

```diff
@@ -33,16 +33,13 @@ RUN pip3 install \
         grpcio-tools \
         itsdangerous \
         rethinkdb \
-        sanic==0.7.0
+        sanic==0.7.0 \
+        tornado==4.5.3 \
```

Comment:
Non-shared dependencies go outside the common block where they were before this change.

Reply:
Ah, Moved them back with a meaningful comment explaining why we do this